ansible_ai_connect/main/views.py

```diff
@@ -115,7 +115,7 @@ class ChatbotView(ProtectedTemplateView):
     template_name = "chatbot/index.html"
 
     permission_classes = [
-        IsAuthenticatedRHEmployee,
+        IsAuthenticatedRHEmployeeOrTestUser,
```

Comment:
Please create this `TestUser` as a separate permission object and use bitwise `|` like https://stackoverflow.com/questions/35557156/how-to-get-or-permissions-instead-of-and-in-rest-framework

Reply:
Yes. That's cleaner. Thanks.